src/eip4824.ts

```diff
@@ -34,31 +34,68 @@ router.get('/:space', async (req, res) => {
 });
 
 router.get('/:space/members', async (req, res) => {
-  let space: any = {};
-
   try {
-    space = await getSpace(req.params.space);
+    const spaceId = req.params.space;
+    const cursor = req.query.cursor || null;  
```

Comment:
When i try http://localhost:3000/api/eip4824/ens.eth/members?cursor=1 i get `Error: NOT_FOUND` error

Reply:
Yes, It is expected behavior, cursor is supposed is to be an Ethereum wallet address, if that wallet address is not of a member or an invalid address it returns NOT_FOUND error.